config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -3809,9 +3806,12 @@ spec:
                             type: string
                           description: Vault annotations that override the Vault annotations set by the cluster-operator. For a list of valid Vault annotations, see https://www.vaultproject.io/docs/platform/k8s/injector/annotations
                           type: object
-                        pathDefaultUser:
+                        defaultUserPath:
                           description: Path in Vault to access a KV secret with the fields username and password for the default user. For example "secret/data/rabbitmq/config".
                           type: string
+                        defaultUserUpdaterImage:
+                          description: 'sidecar container that updates the defaultUser''s password in RabbitMq when it changes in Vault. Additionally, it updates /var/lib/rabbitmq/.rabbitmqadmin.conf (used by rabbitmqadmin CLI). A default image name and version is provided (rabbitmqoperator/admin-password-updater:0.1.1). Only override when the default image version is not appropriate. When set to empty string, it disables the sidecar and passwords are not updated in RabbitMQ. e.g. defaultUserUpdaterImage: ""`'
```

Comment:
typo:
`defaultUser''s password in RabbitMq`

Reply:
Fixed in https://github.com/rabbitmq/cluster-operator/pull/846/commits/9b9a3d0d61f0ed838b279ac767dcdbbdc236a0b2